modules/cpmso-openstack-ts-az-config.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
```

Comment:
This whole block needs rework, if the attempt was to describe https://access.redhat.com/solutions/7024383 - then you need to say that if customers deployed OCP 4.13 clusters with root Volumes without AZ AND compute AZ, then they'll need to update the Machines; It's all explained in the solution. Please have a look at it and let me know what's not clear.

But I'm not sure where you pulled the content bellow, this is totally unrelated to https://access.redhat.com/solutions/7024383.

Reply:
Got that. I will change accordingly.

Reply:
Done. ptal.